examples/nemo/eg3/README.md

```diff
@@ -2,17 +2,21 @@
 
 **Authors:** R. W. Ford and A. R. Porter, STFC Daresbury Lab
 
-This directory contains two example transformation scripts,
-`kernels_trans.py` and `kernels_managed_mem_trans.py`.  The first of these
-demonstrates the use of PSyclone to add OpenACC Kernel and Data directives for
-NEMO code. The second script is similar but is designed to work with the PGI
-compiler's 'managed memory' (`-ta=tesla:managed`) option and thus no attempt is
-made to control data movement to/from the GPU. Note, the transformations are
+This directory contains a relatively simple example transformation script,
+`kernels_trans.py`.  This demonstrates the use of PSyclone to add OpenACC
+Kernel and Data directives to NEMO code. Note, the transformations are
 indicative of what could be done - no claim is made as to the performance of
 the resulting code.
 
-Once you have installed PSyclone, either of the scripts may be supplied to
-PSyclone via the -s option:
+A more sophisticated `kernels_trans.py` script is provided in the
+`../scripts` directory. This is designed to work with the NVIDIA (PGI)
+compiler's 'managed memory' option (`-ta=tesla:managed`) and therefore makes
+no attempt to control data movement to/from the GPU. However, it does
```

Comment:
I'm not sure you should be saying "However, it does add" as if the previous thing is bad. Probably better to to just say "It also adds"

Reply:
I'm not sure I agree that the previous thing is bad but I think "It also adds" is fine so I've gone with it.